src/output-json.c

```diff
@@ -739,6 +739,19 @@ json_t *CreateJSONHeader(const Packet *p, enum OutputJsonLogDirection dir,
         }
     }
 
+    /* mpls */
+    if (p->mpls_idx > 0) {
+        json_t *js_mpls;
+        js_mpls = json_array();
+
+        for (int i = 0; i < p->mpls_idx; i++) {
+            if (unlikely(js != NULL)) {
```

Comment:
checking for js here doesn't really make sense, for 2 reasons:
1. it should check js_mpls instead
2. is should be check outside of the loop